PROHIBITS_OVERWRITING removed from V8 and causing V8Js compile error.

Hey
Getting an issue when compiling V8Js (latest php8 branch) against V8 12.0.267.14

```
/build/v8js/v8js_variables.cc: In function 'void v8js_register_accessors(std::vector<v8js_accessor_ctx*>*, v8::Local<v8::FunctionTemplate>, zval*, v8::Isolate*)':
/build/v8js/v8js_variables.cc:83:185: error: 'PROHIBITS_OVERWRITING' is not a member of 'v8'
   83 | STR_LEN(property_name))), v8js_fetch_php_variable, NULL, v8::External::New(isolate, ctx), v8::PROHIBITS_OVERWRITING, v8::ReadOnly);
```

Is this just as simple as changing `v8::PROHIBITS_OVERWRITING` to `v8::DEFAULT`? It's what we've done for the time being, and it allows everything to compile successfully and all the tests pass - but don't want to be taking any gambles here that it could cause unforeseen issues.

It's been a little while since I've dug into the V8Js code too, so I'd be keen to know what this is for so I can test a specific use case.

Cheers

--- a/v8js_variables.cc
+++ b/v8js_variables.cc
@@ -80,7 +80,7 @@ void v8js_register_accessors(std::vector<v8js_accessor_ctx*> *accessor_list, v8:
         ctx->isolate = isolate;
 
 		/* Set the variable fetch callback for given symbol on named property */
-		php_obj->SetAccessor(V8JS_STRL(ZSTR_VAL(property_name), static_cast<int>(ZSTR_LEN(property_name))), v8js_fetch_php_variable, NULL, v8::External::New(isolate, ctx), v8::PROHIBITS_OVERWRITING, v8::ReadOnly);
+		php_obj->SetAccessor(V8JS_STRL(ZSTR_VAL(property_name), static_cast<int>(ZSTR_LEN(property_name))), v8js_fetch_php_variable, NULL, v8::External::New(isolate, ctx), v8::DEFAULT, v8::ReadOnly);
 
 		/* record the context so we can free it later */
 		accessor_list->push_back(ctx);

